src/Evolution/DiscontinuousGalerkin/SlopeLimiters/MinmodType.cpp

```diff
@@ -9,6 +9,15 @@
 #include "Options/Options.hpp"
 #include "Options/ParseOptions.hpp"
 
+std::ostream& SlopeLimiters::operator<<(
+    std::ostream& os, const SlopeLimiters::MinmodType& minmod_type) {
+  return os << (minmod_type == SlopeLimiters::MinmodType::LambdaPi1
```

Comment:
We've been preferring `switch` cases for the reason that then we get compiler warnings if a new value gets added to the enum and isn't handled in the switch case.